### PR TITLE
Add scroll-to-top button with fade-in and fade-out effect

### PR DESCRIPTION
**Pull request by cu8code**
### Description

added scroll to top button
#90

### Checklist

- [x] I am making a proper pull request, not spam.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#90 this fix was made according to this issue

## Add relevant screens

https://user-images.githubusercontent.com/109351887/194487306-23f9dc4a-e21a-4221-82dc-31b2706d5768.mp4

### CODE REVIEW

**Comment by cu8code** [comment]
Hey, I also added the fade in and fade out effect

https://user-images.githubusercontent.com/109351887/194569063-09f3b687-a4e6-4df1-8ecb-1dcef15ab21c.mp4

**Review by wasimreja — CHANGES_REQUESTED** [review]
resolve the conflict

File: style.css [hunk 1]
```diff
@@ -125,3 +129,60 @@ h1 {
...
+	font-size: 19px;
+	-webkit-transition: all 0.3s ease;
+	-moz-transition: all 0.3s ease;
+	-ms-transition: all 0.3s ease;
+	-o-transition: all 0.3s ease;
+	transition: all 0.3s ease;
+}
+
+#return-to-top:hover {
+	background: rgb(73, 122, 255);
```
Comment (wasimreja) [inline style.css:170]: change background to rgb(208, 197, 197)

File: index.html [hunk 2]
```diff
@@ -54,6 +54,9 @@ <h1 class="title" style="font-size: 7em">
     <div id="results" class="results container">
     </div>
   </section>
+  <a href="#" id="return-to-top" class="special1" > 
+    <i class="fa fa-arrow-up" />
```
Comment (wasimreja) [inline index.html:58]: remove this code

Comment (wasimreja) [inline index.html:58]: remove this code

**Comment by cu8code** [comment]
@wasimreja I made the required changes 🙂

**Pull request merged**